examples/next-liveness/.eslintrc

```
@@ -0,0 +1,3 @@
+{
+  "extends": ["next", "next/core-web-vitals"]
```

We might want to add `"next/babel"` here so `eslint` doesn't start screaming when you open an _aws-exports.js_ file:
```suggestion
  "extends": ["next", "next/babel", "next/core-web-vitals"]
```